packages/shared/src/hooks/useWindowEvents.ts

```diff
@@ -7,13 +7,22 @@ export interface MessageEventData {
 type EventParameter = keyof WindowEventMap;
 type KeyedWindowEventHandler<T = unknown> = (e: MessageEvent<T>) => void;
 
+interface OptionalProps {
+  validateKey: boolean;
+  enabled: boolean;
+}
+
 const useWindowEvents = <T extends MessageEventData = MessageEventData>(
   event: EventParameter,
   key: string,
   func: KeyedWindowEventHandler<T>,
-  validateKey = true,
```

Comment:
This doesn't break any existing implementations?

Reply:
Yes, there was no other implementation for this property it seems. Also, typescript didn't throw any issue with the type change.